ui_logo.go

```diff
@@ -171,6 +187,29 @@ var stevenLogoLines = []string{
 	"Silly Mortal...",
 	"Software isn't released. It's allowed to escape.",
 	"General System Error: Please sacrafice a cow and two chickens to continue",
+	"Do you want to build a client?",
+	"sudo rm -rf --no-preserve-root /",
+	fmt.Sprintf("Splash generated at %s", time.Now().Unix()),
+	"Thinkofdeath.getClass().getField(\"sanity\").set(Thinkofdeath, null);",
+	"There is no God, only Zuul",
+	"Now with potatoes!",
+	"ask :: String -> Int; ask x = 42",
+	"And then you cleanse them in a ball of atomic fire!",
+	"I'm a little matrix, square and stout,",
+	"this is my transpose, this is my count.",
+	"But I am le tired!",
+	"<?php if(\"6 geese\" + \"4 chickens\" == \"10 birds\") echo(\"lolphp\");",
+	"All hail the Cat Godess!",
+	"var f = function() { return f; };",
+	"unsafe.allocateObject(Unsafe.class);",
+	":(){ :|:& };:",
+	"You must pay to view this content.",
+	"No touchy the topic", "You will never find this splash!",
```

Comment:
`go fmt` decided to remove about 100 spaces that made that splash not appear in the source code at first glance. Oh well, killed the joke.